ReSharper.FSharp/src/FSharp/FSharp.Psi.Features/src/Injected/FSharpInjectionUtil.fs

```diff
@@ -8,6 +8,11 @@ open JetBrains.ReSharper.Plugins.FSharp.Psi.Features.Util.FSharpMethodInvocation
 open JetBrains.ReSharper.Plugins.FSharp.Psi.Impl
 
 let getAttributesOwner (expr: IFSharpExpression) attributeNames =
+    let getAttributesOwner (declaredElement: IDeclaredElement) =
```

Comment:
This name is somewhat misleading. With this change there's now a `getAttributesOwner` function inside `getAttributesOwner` one that has a different input and tries handle accessor declared elements in a special way. If it's only about trying to get an accessor owners, let's maybe have this stated in the name?

Reply:
Maybe we could rename the external function to `locateAttributeOwner`? It really does a search based on an arbitrary expression, while `getAttributesOwner (declardelement: IDeclaredElement)` really gets the attributes owner for declared element, which is the property for the accessor.